Source/Core/DolphinQt/MainWindow.cpp

```diff
@@ -165,6 +178,33 @@ void DMainWindow::OnOpen()
 		StartGame(filename);
 }
 
+void DMainWindow::OnBrowse()
+{
+	std::string path = ShowFolderDialog().toStdString();
+	std::vector<std::string>& iso_folder = SConfig::GetInstance().m_ISOFolder;
+	if (!path.empty())
+	{
+		auto itResult = std::find(
+				iso_folder.begin(),
+				iso_folder.end(), path);
```

Comment:
Should spaces be used over tabs here as well? If so, CGameListCtrl::BrowseForDirectory() should be altered as it uses the same indentation on its iterator (I lifted most of that method for this one).